e2e-tests/manifest

```diff
@@ -36,20 +36,19 @@ sed '/.*- protocol/d; s/.*- containerPort/&\n&/; s/- containerPort/- protocol/'
 crd_file=$(mktemp)
 yq r "$wip_crd" >"$crd_file"
 
-echo "---" >>"$crd_file"
-yq r "deploy/crds/pxc.percona.com_perconaxtradbclusterbackups.yaml" >>"$crd_file"
-echo "---" >>"$crd_file"
```

Comment:
we need to separate yaml docs with `---`

Reply:
@egegunes files in `deploy/crds` folder are already generated with `---` at the top